### ISSUE TITLE
Tests on travis fail because of puli

### ISSUE THREAD

**Issue by gossi** [puli/issues#176]
I realized, tests on travis are failing because of puli. Here is a travis log: https://travis-ci.org/keeko/core/jobs/102379958

Seems like that happens because a puli-cli is missing?

**Comment by webmozart** [comment]
Hi @gossi :) Did you try adding puli/cli as dev dependency?

**Comment by gossi** [comment]
Nope, and doing so would feel wrong.

**Comment by webmozart** [comment]
@gossi Then you need to adjust your build script to install Puli by running `curl https://puli.io/installer | puli`.

**Comment by gossi** [comment]
Nope, wrong, too. Every I need to do is a workaround to cope with a puli problem. My tests run fine, when everything is installed, yet puli prevents this install. I expect puli to be clever enough to skip it's own installation mechanism, when its required dependencies are not present. Well, at least that it doesn't break other's code with a `Fatal Error` but degrade gracefully with a warning or something equivalent.

**Comment by webmozart** [comment]
Ah, I just saw the fatal error in the log. That's a bug indeed. The install should proceed after the warning.

**Comment by Spomky** [comment]
Hi,

Is there any news regarding this issue.
It does not work for me as well (see https://travis-ci.org/Spomky-Labs/OAuth2ServerBundle/jobs/125765205).
